### PR TITLE
[FR] Update `Ranking criteria`

### PR DESCRIPTION
**Pull request by DoryanR**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review) > **No contributor**

### CODE REVIEW

File: wiki/Ranking_criteria/fr.md [hunk 1]
```diff
@@ -1,308 +1,302 @@
...
 
-- Aucun outil ne va remplacer la nécessité de lire cette page ou de vérifier manuellement.
-- Soyez critique à l'égard de ce qu'ils signalent et ne les suivez pas aveuglément.
+- Aucun outil ne remplace entièrement la nécessité de lire cette page ou de vérifier manuellement.
+- Soyez vigilant à l'égard de ce qu'ils indiquent et ne les suivez pas aveuglément.
 - Ils existent pour vous aider, pas pour vous remplacer.
 
 ## Glossaire
 
 ### Termes généraux
```
Comment (DoryanR) [inline wiki/Ranking_criteria/fr.md:23]:
```suggestion
### Termes courants
```

File: wiki/Ranking_criteria/fr.md [hunk 2]
```diff
@@ -1,308 +1,302 @@
----
-outdated_since: 2d4959ec9ad9cb438363d7bf8e8220687a899602
-outdated_translation: true
----
-
 # Critères de classement
 
 *Pour les critères de classement spécifiques aux [modes de jeu](/wiki/Game_mode), voir : [osu!](osu!), [osu!taiko](osu!taiko), [osu!catch](osu!catch), et [osu!mania](osu!mania).*
 
 Cet ensemble de **critères de classement** définit les [règles et directives](#termes-généraux) que les [beatmaps](/wiki/Beatmap) doivent suivre pour progresser dans la [procédure de classement des beatmaps](/wiki/Beatmap_ranking_procedure).
```
Comment (DoryanR) [inline wiki/Ranking_criteria/fr.md:5]:
```suggestion
Cet ensemble de **critères de classement** définit les [règles et directives](#termes-courants) que les [beatmaps](/wiki/Beatmap) doivent suivre pour progresser dans la [procédure de classement des beatmaps](/wiki/Beatmap_ranking_procedure).
```

**Review by TicClick — DISMISSED** [review]
merging (no native review for one week or longer)

**Comment by TicClick** [comment]
(if anything, I ignored https://github.com/ppy/osu-wiki/pull/10260#issuecomment-1774109971 to avoid keeping the translation open for another extended period of time)

**Review by TicClick — APPROVED** [review]

**Pull request merged**